dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -13,14 +13,19 @@ import { enhanceTreats } from './enhanceTreats';
 import { groupCards } from './groupCards';
 
 const FORBIDDEN_CONTAINERS = [
-	'Palette styles new do not delete',
-	'Palette styles',
 	'culture-treat',
 	'newsletter treat',
 	'qatar treat',
 ];
 const isSupported = (collection: FECollectionType): boolean =>
-	!FORBIDDEN_CONTAINERS.includes(collection.displayName);
+	!(
+		FORBIDDEN_CONTAINERS.includes(collection.displayName) ||
+		collection.curated.some(
+			(card) =>
+				card.properties.embedUri ===
+				'https://content.guardianapis.com/atom/interactive/interactives/2022/03/29/fronts-container-colours/default',
```

Comment:
It might help future travellers by creating a named const?

You'll have a better name but something like:

```suggestion
				PALETTE_STYLES_URI,
```

Reply:
That's a nice idea! I'll make the change